be/app/Http/Controllers/LessonController.php

```diff
@@ -61,9 +65,20 @@ public function show($id)
      * @param  int  $id
      * @return \Illuminate\Http\Response
      */
-    public function update(Request $request, $id)
+    public function update(StoreLessonRequest $request, $id)
     {
-        //
+        $request->validated();
+
+        $lesson = Lesson::find($id);
+        $lesson->title = $request->title;
+        $lesson->description = $request->description;
+        $lesson->save();
+
+        return response([
+            'status' => 200,
```

Comment:
- if the operation in a function involves modification of data e.g. `create`, `update`, `delete`, etc. Please use status code `201`. Status `200` is for fetching data only.